fastlane/lib/fastlane/fast_file.rb

```diff
@@ -217,7 +213,8 @@ def import(path = nil)
       actions_path = File.join(File.expand_path("..", path), 'actions')
       Fastlane::Actions.load_external_actions(actions_path) if File.directory?(actions_path)
 
-      collector.did_launch_action(:import)
+      action_launched('import_from_git')
```

Comment:
This is not `import_from_git`, right? It's `import` for local Fastfiles

Reply:
copy 🍝